packages/now-next/src/index.ts

```diff
@@ -1184,7 +1185,7 @@ export const build = async ({
       entryPath,
       entryDirectory,
       dynamicPages,
-      false,
+      meta.isDev,
```

Comment:
```suggestion
      false
```
This code path should never be reached with `vercel dev` as we return above on line 299 https://github.com/vercel/vercel/blob/0f5f99e6678fd1127947cf43a531d0976bc6bca1/packages/now-next/src/index.ts#L299